ninja/testing/client.py

```diff
@@ -104,12 +104,17 @@ def _resolve(
         for url in self.urls:
             match = url.resolve(url_path)
             if match:
-                request = self._build_request(method, path, data, request_params)
+                request = self._build_request(method, path, data, request_params, match)
                 return match.func, request, match.kwargs
         raise Exception(f'Cannot resolve "{path}"')
 
     def _build_request(
-        self, method: str, path: str, data: Dict, request_params: Any
+        self,
+        method: str,
+        path: str,
+        data: Dict,
+        request_params: Any,
+        resolver_match: Any,
```

Comment:
this could probably be better typed, but honestly, since it's used internally and not exposed as the external testing client's API, I was personally fine with `Any` and be done with it.